frontend/src/redux/reducers/otherUserDataReducer.tsx

```diff
@@ -67,6 +68,20 @@ export const otherUserDataReducer = (
         error: action.payload,
         success: false
       };
+    case OtherUserDataActionTypes.UPDATE_USER_DATA_AND_BAN_LOADING:
+      return {
+        loading: false,
+        error: null,
+        data: initialState.data,
+        success: false
+      };
+    case OtherUserDataActionTypes.UPDATE_USER_DATA_AND_BAN_SUCCESS:
+      return {
+        loading: false,
+        error: null,
+        data: action.payload,
+        success: true
+      };
     default:
```

Comment:
i think 'onError' case should be added here

Reply:
done